src/odemis/driver/andorshrk.py

```diff
@@ -295,47 +307,55 @@ class Shamrock(model.Actuator):
-          specified, the standard hardware limits are used.
-          For now it *only* works for the focus axis.
-        drives_shutter (list of float): flip-out angles for which the shutter
-          should be set to BNC (external) mode. Otherwise, the shutter is left
-          opened.
-        dependencies (None or dict str -> HwComponent): if the key starts with
-          "led_prot", it will set the .protection to True any time that the
-          slit leds could be turned on.
+        :param device (0<=int or str): if int, device number, if str serial number or
+        "fake" to use the simulator
+        :param camera (None or AndorCam2): Needed if the connection is done via the
+        I²C connector of the camera.
+        :param accessory: if "slitleds", then a TTL signal will be set to
+        high on line 1 whenever one of the slit leds might be turned on.
+        :param slitleds_settle_time (0 <= float): duration wait before (potentially)
+        turning on the slit leds. Useful to delay the move after the slitleds
+        interlock is set.
+        :param slits (None, or dict int -> str, or dict int -> [str]): names of each slit,
+        for 1 to 4: in-side, in-direct, out-side, out-direct
+        Append "force_max" for a slit which requires to move to the maximum
+        value before going to the requested position. This is a workaround for
+        proper movement when the slit's reference switch doesn't work.
+        :param iris: for each iris (0=direct, 1=side) -> axis name.
+        :param bands (None or dict 1<=int<=6 -> 2-tuple of floats > 0 or str):
+        wavelength range or name of each filter for the filter wheel from 1->6.
+        Positions without filters do not need to be defined.
+        :param rng (dict str -> (float, float)): the min/max values for each axis.
+        They should within the standard hardware limits. If an axis is not
+        specified, the standard hardware limits are used.
+        For now, it *only* works for the focus axis.
+        :param fstepsize (0<float): size of one step on the focus actuator. Not very
+        important, mostly useful for providing to the user a rough idea of how
+        much the image will change after a move.
+        :param drives_shutter: flip-out angles for which the shutter should be set
+        to BNC (external) mode. Otherwise, the shutter is left opened.
+        :param dependencies (None or dict str -> HwComponent): if the key starts with
+        "led_prot", it will set the .protection to True any time that the
+        slit leds could be turned on.
         :param check_move: Name of the axis -> bool: If True (default), after move, raise an error if
         that position is not the expected one. If False, if the position is not the expected one,
         just log a warning. Note: for now, only some axes are actually checked (flipper and band)
+        :param inverted: Must be None (or not passed), as it is not allowed to invert the axes.
```

Comment:
param inverted is not an explicit argument, is it required to be included here? You could also move it ( as a non-argument description) to line 366.

Reply:
It's not explicit in the signature, but it's part of the Actuator signature, and it's handled explicitly in the code. So I think it's worthy to be placed in the docstring.